Bottom: f0700d283fd8f7d62c707b01bff0874096c7794f
Top:    6ae5747e77beae4491a6af485576b34e9561eabb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:53:25 +0000

Disable small edge EPSILON hack for computing edge normals


---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index ee8938b..5781a61 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -231,7 +231,7 @@ object3d_to_step_body_fragment (step_file *step,
           dir_y = dv->y - ov->y;
           dir_z = dv->z - ov->z;
 
-#if 1
+#if 0
           /* XXX: This avoids the test file step_outline_test.pcb failing to display properly in freecad when coordinates are slightly rounded */
           if (dir_x < EPSILON && -dir_x < EPSILON &&
               dir_y < EPSILON && -dir_y < EPSILON &&
